### ISSUE TITLE
"Cannot find the pagebotosx module"

### ISSUE THREAD

**Issue by simoncozens** [PageBot/PageBotCocoa#7]
```shell
% sudo -H pip3 install pagebotosx
...
```

```Python console
>>> import pagebotosx

>>> from pagebot.document import Document
Cannot find the pagebotosx module.
Using Flat instead of DrawBot.

>>> pagebotosx.__path__
['/usr/local/lib/python3.7/site-packages/pagebotosx']
>>> pagebotosx.__version__
'0.1.4-alpha'

>>> pagebot.__version__
'0.9.9-alpha'

**Comment by michielkauwatjoe** [comment]
Hi, do you have the correct drawbot installed?

    pip install git+https://github.com/typemytype/drawbot.git#egg=drawbot

(`pip3` if python 2 is your default)

**Comment by simoncozens** [comment]
I may have missed that it the instructions - where does it say I need to do that?

**Comment by simoncozens** [comment]
OK, getting further now:

```
    self.context.setSize(self.w, self.h)
AttributeError: 'NoneType' object has no attribute 'setSize'
```

I'll try installing pagebot from github master.

**Comment by michielkauwatjoe** [comment]
> I may have missed that it the instructions - where does it say I need to do that?

It should be done automatically since it's in `requirements.txt` (needs some more testing apparently).

**Comment by simoncozens** [comment]
Now I need to install pysketch (but not the module with the same name on pypi). And also install pagebotosx from github master. Now I can run E00_HelloWorld.py, but I get:

```
    st.tracking(tracking/10) # Flat tracking if %, BabelString is 1/1000em
AttributeError: 'strike' object has no attribute 'tracking'
```

**Comment by michielkauwatjoe** [comment]
> Now I need to install pysketch (but not the module with the same name on pypi). And also install pagebotosx from github master. Now I can run E00_HelloWorld.py, but I get:
> 
> ```
>     st.tracking(tracking/10) # Flat tracking if %, BabelString is 1/1000em
> AttributeError: 'strike' object has no attribute 'tracking'
> ```

From the look of it it's trying to run the Flat context instead of DrawBot, which should also give decent output. However, we're still working towards a pull request for tracking in Flat, so in the meantime you need to install our fork:

https://github.com/PageBot/flat

**Comment by simoncozens** [comment]
OK, getting somewhere now! Strange output but at least some output. Thanks!

**Comment by michielkauwatjoe** [comment]
@simoncozens yes the output quality may vary, I'm still debugging many of the examples. We've done some restructuring of the string representation (BabelString) to synchronize DrawBot (OSX) and Flat (OSX + Linux) support. Most of the examples used to be DrawBot only, we're now expanding them to give the correct results for both contexts. They should be stable in a couple of weeks hopefully.

**Comment by simoncozens** [comment]
Cool. I hope this goes well, but to be honest after this start, I've now given up and already got what I wanted to do working in another tool. Leaving this issue open as there's still some problem with installing all the dependencies.

**Comment by michielkauwatjoe** [comment]
Okay, thank you for your input!
